drivers/eks/eks_driver.go

```diff
@@ -42,67 +42,80 @@ const (
 
 var amiForRegionAndVersion = map[string]map[string]string{
 	"1.17": {
-		"us-east-2":      "ami-044cba456c7d6a2fe",
-		"us-east-1":      "ami-04125ecea1c9b3b27",
-		"us-west-2":      "ami-037843f6aeb12e236",
-		"ap-east-1":      "ami-092dc7701bd03af3e",
-		"ap-south-1":     "ami-0e51866c4b1e01c77",
-		"ap-northeast-1": "ami-095dcd341e28f2599",
-		"ap-northeast-2": "ami-06a3ded9b6c463c6f",
-		"ap-southeast-1": "ami-0365ad3c8cfd3cb4c",
-		"ap-southeast-2": "ami-0e55c01aa1b1cf3cd",
-		"ca-central-1":   "ami-00a3415fa128f17c5",
-		"eu-central-1":   "ami-0c28233a2bd46bd3e",
-		"eu-west-1":      "ami-0cb5f54d0d7b2ed21",
-		"eu-west-2":      "ami-05f8e36acad8edc61",
-		"eu-west-3":      "ami-0d6f4cc928f18710e",
-		"eu-north-1":     "ami-043dbb11ff9b5a350",
-		"me-south-1":     "ami-0209971c7465bb090",
-		"sa-east-1":      "ami-0ff3ff7ab99c06946",
```

Comment:
We're missing at least 3 regions that might not have been on the list from the script, but we can't just drop them. 

Can we find the AMI manually from this list? If you don't have access, don't delete them, but leave them as unchanged. 

https://docs.aws.amazon.com/eks/latest/userguide/eks-optimized-ami.html

This is super hard to review cause the regions don't line up so I don't even know what regions are missing.

Reply:
I can't access some of them but i will make sure there is a record for them.